app/components/chips/base_chip.tsx

```diff
@@ -11,17 +11,17 @@ import {nonBreakingString} from '@utils/strings';
 import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
 import {typography} from '@utils/typography';
 
+import {CHIP_BOTTOM_MARGIN, CHIP_HEIGHT} from './constants';
+
 type SelectedChipProps = {
-    id: string;
-    text: string;
-    extra?: React.ReactNode;
-    onRemove: (id: string) => void;
+    onPress: () => void;
     testID?: string;
-    containerStyle?: StyleProp<ViewStyle>;
+    showXButton?: boolean;
```

Comment:
Maybe rename this to `showRemoveOption` or something similar

Reply:
It's not exactly a button, right?  It's an (x) inside the user chip? :P